_newsletters/2019-05-01-may-2019.md

```diff
@@ -0,0 +1,106 @@
+
+### How Does it work?
+
+The sitelimit parameter overrides the <a href="https://search.gov/manual/domains.html">Domains list</a>. However, to work, the paths included have to be within the scope of the Domains list in the admin center for the site.
+
+Simply add the following to the URL for the search page:
+
+`&sitelimit=www.exampledomain.gov/limittothisfolder`
+
+Multiple paths can be combined with a +, for example:
+
+`&sitelimit=www.exampledomain.gov/limittothisfolder+folderarchive.exampledomain.gov`.
+
+### How to add it
+
+Sitelimit parameters can be added via an input in the search box
+
+```<input type="hidden" name="sitelimit" id="sitelimit" value="www.exampledomain.gov/limittothisfolder">```
+
+or manually by typing it at the end of the query url.
+
+## User Account Changes Coming
+
+### Automatic Account Deactivation
+
+In the next few months we are going to implement automatic account deactivation for users that are **inactive for 90 days** or more. To ensure your account stays active, we ask you to please login today, and come back and visit us often!
+
+If in the future you need to have your access reinstated, <a href="mailto:search@gsa.gov">please email us</a>.
+
+## DEPRECATION NOTICE
+
+### Legacy Results Page Will be Decommissioned June 28
+
+In 2014 we introduced a mobile friendly responsive results page, while allowing some agencies to continue using the legacy results page. Due to a security vulnerability, non responsive design issues, and budget constraints, we will no longer offer the legacy results page.
+
+### Important Dates
+
+By COB **June 28th**, all sites still using this legacy system will be switched to our mobile friendly layout. If you would like to switch sooner than that, we would be happy to work with you. The week leading up to June 28, we will begin working through all the sites left to convert, even if we have not heard from you: we’ll locate your agency’s basic logo, upload it to the system to serve as your header logo, and on June 28 we will switch your layout.
+
+To view what your results page will look like without any changes, please add your site handle to the link below:
+
+https://search.usa.gov/search/?affiliate=YOURSITEHANDLEGOESHERE&amp;utf8=%E2%9C%93&amp;m=true
+
+### Make the Switch!
+
+Here are some helpful steps to customize your search results page and match your site’s brand before **June 28, 2018:**
+
+**Create a new site**
+
+Use the Add Site button at the top of the Admin Center to create a new site. Here is where you can play around with the look and feel before going live.
+
+<a href="https://search.gov/manual/brand.html?utm_source=hs_email&amp;utm_medium=email&amp;_hsenc=p2ANqtz-8asXLD8kW1r6huIYx7xdkq4QtNA47llNtSUy9jkEVwU5dRiMUYsSnNV0lCI1aMs138m2T3">**Define your color scheme on the Fonts &amp; Colors page**
```

Comment:
where does this anchor tag end?

Reply:
will fix, but seems to be displaying right: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/header-newsletter-updates/about/updates/newsletters/may-2019.html